### PR TITLE
Triggering GitHub Actions from forks.

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: practices/ci/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+```yaml
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+```
+which will trigger only `pull_request` event for pull requests, but not `push` event when a PR is created.
+
+Another example, when using [`bors`][bors] one SHOULD configure GitHub Actions with:
```
[the remark this answers is not in the document]
Comment (angerman) [reply practices/ci/README.md:17]: I agree that we want to reference ideas, and then maybe mention specific instances. I think this feature is generally called merge train, and both bors, and mergify are instances of this.

File: practices/ci/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## GitHub Actions
+
+If you are using GitHub Actions it SHOULD be configured so that pull requests from forks will trigger it.
```
[the remark this answers is not in the document]
Comment (angerman) [reply practices/ci/README.md:3]: How would this work if we added shared GitHub runners? We probably do not want to have anyone be able to open PRs that run arbitrary code (bitcoin mining), on shared runners. Especially with poor isolation, on at least macOS, PRs don't even run properly insulated from each other :(

[the remark this answers is not in the document]
Comment (coot) [reply practices/ci/README.md:3]: GitHub has a config option in which disables CI runners for first time contributors. It's easy to go around it though (e.g. submit a PR which just fixes a typo).  We could be more strict and require it on every CI run for non contributors, [ref](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks).

File: practices/ci/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (coot) [reply practices/ci/README.md]: Expanded the section, did you know that GitHub has [merge queues](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue) (currently in beta).

**Comment by coot** [comment]
@michaelpj, @abailly-iohk is there anything else you'd like me to address?

File: practices/ci/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (michaelpj) [reply practices/ci/README.md:3]: This comment still stands about the SHOULD not fitting outside of policies, but not a big deal

**Review by michaelpj — APPROVED** [review]

**Pull request merged**